docs/design/FeatureSwitches.md

```diff
@@ -13,6 +13,7 @@ The following switches are toggled for applications running on Mono for `TrimMod
 | _MauiBindingInterceptorsSupport | Microsoft.Maui.RuntimeFeature.AreBindingInterceptorsSupported | When disabled, MAUI won't intercept any calls to `SetBinding` methods and try to compile them. Enabled by default. |
 | MauiEnableXamlCBindingWithSourceCompilation | Microsoft.Maui.RuntimeFeature.XamlCBindingWithSourceCompilationEnabled | When enabled, MAUI will compile all bindings, including those where the `Source` property is used. |
 | MauiHybridWebViewSupported | Microsoft.Maui.RuntimeFeature.IsHybridWebViewSupported | Enables HybridWebView, which makes use of dynamic System.Text.Json serialization features |
+| MauiEnableMetrics | System.Diagnostics.Metrics.Meter.IsSupported | Enables MAUI�s metrics instrumentation via `Meter`. Metrics are on by default; set to `false` to disable.   |
```

Comment:
Is the property actually `$(MetricsSupport)`?

* https://github.com/dotnet/sdk/blob/a631dd38ec1933cbd8909e252ef51f7d6bf4465c/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L549-L552

I don't think we'd want to introduce a new property when they already have one.